Parsers/ASimWebSession/Parsers/vimWebSessionPaloAltoCortexDataLake.yaml

```diff
@@ -0,0 +1,241 @@
+  This ASIM parser supports normalizing Palo Alto Cortex Data Lake logs to the ASIM Web Session normalized schema. These events are captured through the Palo Alto Networks CDL data connector that ingests CDL logs into Microsoft Sentinel.
+ParserName: vimWebSessionPaloAltoCortexDataLake
+EquivalentBuiltInParser: _Im_WebSession_PaloAltoCortexDataLake
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: srcipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: ipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: url_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: httpuseragent_has_any
+    Type: dynamic 
+    Default: dynamic([])
+  - Name: eventresultdetails_in
+    Type: dynamic 
+    Default: dynamic([])
+  - Name: eventresult
+    Type: string
+    Default: '*'
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let EventSeverityLookup = datatable (LogSeverity: string, EventSeverity: string)
+      [
+      "0", "Low",
+      "1", "Low",
+      "2", "Low",
+      "3", "Low",
+      "4", "Medium",
+      "5", "Medium",
+      "6", "Medium",
+      "7", "High",
+      "8", "High",
+      "9", "High",
+      "10", "High"
+  ];
+  let EventLookup=datatable(DeviceAction:string, DvcAction:string,EventResult:string,EventResultDetails:string)
+    [
+        "alert", "Allow", "Success","200", 
+        "allow", "Allow", "Success", "200",
+        "continue", "Allow", "Success", "200",
+        "override", "Allow", "Success", "200",
+        "block-continue", "Allow", "Partial", "200",
+        "block-url", "Deny", "Failure", "503",
+        "block-override", "Deny", "Failure", "302",
+        "override-lockout", "Deny", "Failure","503",
+        "reset client", "Reset Source", "Failure", "503",
+        "reset server", "Reset Destination", "Failure", "503",
+        "reset both", "Reset", "Failure", "503",
+        "deny", "Deny", "Failure", "503",
+        "drop", "Drop", "Failure", "503",
+        "drop ICMP", "Drop ICMP", "Failure", "503"
+    ];
+  let parser = (starttime: datetime=datetime(null),
+      endtime: datetime=datetime(null),
+      srcipaddr_has_any_prefix: dynamic=dynamic([]),
+      ipaddr_has_any_prefix: dynamic=dynamic([]),
+      url_has_any: dynamic=dynamic([]),
+      httpuseragent_has_any: dynamic=dynamic([]),
+      eventresultdetails_in: dynamic = dynamic([]),
+      eventresult: string='*',
+      disabled: bool=false) {
+      let src_or_any = set_union(srcipaddr_has_any_prefix, ipaddr_has_any_prefix);
+      CommonSecurityLog
+      | where not(disabled)
+          and (isnull(starttime) or TimeGenerated >= starttime)
+          and (isnull(endtime) or TimeGenerated <= endtime)
+          and DeviceVendor == "Palo Alto Networks" and DeviceProduct == "LF"
+          and DeviceEventClassID == "THREAT" and Activity == "url"
```

Comment:
Can we map Activity to [EventOriginalSubType](https://learn.microsoft.com/en-us/azure/sentinel/normalization-common-fields#eventoriginalsubtype)

Reply:
Done